### PR TITLE
Move newly added variables and edx_django_service variables to app repo

### PR DESCRIPTION
**Pull request by syedimranhassan**
## Description

TODO: Include a detailed description of the changes in the body of the PR

### CODE REVIEW

File: registrar/settings/base.py [hunk 1]
```diff
@@ -263,8 +273,8 @@
 # END AUTHENTICATION CONFIGURATION
 
 # Other service locations
-LMS_BASE_URL = 'replace-me'
```
Comment (kdmccormick) [inline registrar/settings/base.py]: Why change these to `None`?

File: registrar/settings/base.py [hunk 2]
```diff
@@ -278,7 +288,7 @@
 SEGMENT_KEY = None
 
 # Publicly-exposed base URLs for service and API
-API_ROOT = 'http://replace-me/api'
```
Comment (kdmccormick) [inline registrar/settings/base.py:281]: As everything else in here is based on `127.0.0.1:8000`, can you make this `http://127.0.0.1:8000/api`?

File: registrar/settings/base.py [hunk 3]
```diff
@@ -290,3 +300,25 @@
...
+    'DEFAULT_FILE_STORAGE': 'django.core.files.storage.FileSystemStorage',
+    'MEDIA_ROOT': MEDIA_ROOT,
+    'MEDIA_URL': MEDIA_URL
+}
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+    }
+}
+SOCIAL_AUTH_EDX_OIDC_KEY = 'registrar-key'
```
Comment (kdmccormick) [inline registrar/settings/base.py]: As I understand it, edX OpenIDConnect is deprecated, so would it be safe for us to nix the `SOCIAL_AUTH_EDX_OIDC_` settings entirely?

File: registrar/settings/base.py [hunk 4]
```diff
@@ -290,3 +300,25 @@
 REGISTRAR_SERVICE_USER = 'registrar_service_user'
 SESSION_EXPIRE_AT_BROWSER_CLOSE = False
 STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
+CORS_ORIGIN_WHITELIST = []
+CSRF_TRUSTED_ORIGINS = []
+MEDIA_STORAGE_BACKEND = {
```
Comment (kdmccormick) [inline registrar/settings/base.py]: `MEDIA_STORAGE_BACKEND` is not a real Django setting; it's just something we use in our configuration pipeline, so you can remove this dictionary.

File: registrar/settings/base.py [hunk 5]
```diff
@@ -290,3 +300,25 @@
 REGISTRAR_SERVICE_USER = 'registrar_service_user'
 SESSION_EXPIRE_AT_BROWSER_CLOSE = False
 STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.StaticFilesStorage'
+CORS_ORIGIN_WHITELIST = []
```
Comment (kdmccormick) [inline registrar/settings/base.py]: Can you put `CORS_ORIGIN_WHITELIST` with the other CORS settings (line ~76)?

**Review by kdmccormick — CHANGES_REQUESTED** [review]
Generally looks good, just a few comments.

File: registrar/settings/base.py (hunk 3 above)
Comment (coryleeio) [reply registrar/settings/base.py]: We're unsure - if you say it is we can remove it.  We are just merging the default values from configuration into this repo in preparation to delete them from the configuration repo.

If you say it is okay to remove it we can remove it, let us know if you want it removed.

**Review by coryleeio — APPROVED** [review]
This is good on my end - but we should address Kyle's feedback before merging.

File: registrar/settings/base.py (hunk 3 above)
Comment (kdmccormick) [reply registrar/settings/base.py]: Sounds good. I did some digging... you're good to drop all the `SOCIAL_AUTH_EDX_OIDC_*` settings @syedimranhassan

**Pull request merged**